### ISSUE TITLE
Feature Request: Button Position 

### ISSUE THREAD

**Issue by stphnwlkr** [itmaybejj/editoria11y#24]
It seems like every add-on puts its interactive button in the lower right corner. It would be helpful to position the button on the left or set the position with a percentage -- right and bottom.

**Comment by itmaybejj** [comment]
Indeed. I have often thought about this, because I have moved it on my own sites several times using CSS in my theme, e.g.:
`ed11y-element-panel {
  right: auto;
  left: 1%;
}`

I _have_ to rewrite some of this logic and add some related parameters in the [ContentEditable branch](https://editoria11y.princeton.edu/next/), so I will consider wrapping this into that in the fall.

**Comment by itmaybejj** [comment]
@stphnwlkr -- Done. 2.3.0 has parameters to [pin to the left, apply a horizontal and vertical offset, and automatically avoid other widgets by selector](https://github.com/itmaybejj/editoria11y/blob/2.3.x/js/ed11y.js#L84).
